shakenfist/external_api/app.py

```diff
@@ -1275,18 +1275,11 @@ def post(self, netblock=None, provide_dhcp=None, provide_nat=None, name=None,
                 n.create()
```

Comment:
To avoid code duplication what do you think about the network node queuing the network change?
Also removes the unnecessary check of network uuid validity.
You will also notice that db.enqueue() snuck into the lock context when it doesn't need the lock.

Reply:
That's a good idea. Done.